Rubberduck.Core/UI/Controls/NumberPicker.xaml.cs

```diff
@@ -22,9 +27,26 @@ public int NumValue
             }
         }
 
-        public int MinNumber { get; set; } = int.MinValue;
 
-        public int MaxNumber { get; set; } = int.MaxValue;
+        public int MinNumber
+        {
+            get => (int)GetValue(MinNumberProperty);
+            set
+            {
+                SetValue(MinNumberProperty, value);
+                OnPropertyChanged(new DependencyPropertyChangedEventArgs(MinNumberProperty, NumValue - 1, NumValue));
+            }
+        }
+
+        public int MaxNumber
+        {
+            get => (int)GetValue(MaxNumberProperty);
+            set
+            {
+                SetValue(MaxNumberProperty, value);
+                OnPropertyChanged(new DependencyPropertyChangedEventArgs(MaxNumberProperty, NumValue - 1, NumValue));
```

Comment:
I don't think `NumValue - 1` is accurate on either property... the min and max values determine the lower and upper bounds of values that can legally be assigned to `NumValue` - neither are involved in the incrementing/decrementing.

````csharp
set
{
    var oldValue = MaxNumber; // ditto MinNumber
    SetValue(MaxNumberProperty, value); // ditto MinNumberProperty
    OnPropertyChanged(new DependencyPropertyChangedEventArgs(MaxNumberProperty, oldValue, value));
}
````

That said - and it's probably out of scope for this PR, but I'd rename them `MinValue` and `MaxValue`, and `NumValue` to `Value` (unless it clashes with an existing member?)

Reply:
Oops - copy paste error. Fixed (deferring the rename - no clue where all the uses are).